### PR TITLE
Added displayName #208

### PR DESCRIPTION
**Pull request by rigazilla**
added missing displayName. #208

### CODE REVIEW

File: deploy/olm-catalog/infinispan-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -205,10 +205,14 @@ spec:
             displayName: Extra JVM Options
             path: container.jvmOptionsAppend
           - description: Name of the secrect that contains auth info
+            displayName: Endpoint SecretName
```
Comment (Crumby) [inline deploy/olm-catalog/infinispan-operator.clusterserviceversion.yaml]: SecretName -> Secret Name?

File: deploy/olm-catalog/infinispan-operator.clusterserviceversion.yaml [hunk 2]
```diff
@@ -205,10 +205,14 @@ spec:
...
             path: security.endpointSecretName
           - description: Select how encrption certificates are provided [Secret,Service]
+            displayName: Endpoint Encryption Type
             path: security.endpointEncryption.type
           - description: Platform service that serves certificates (openshift.io/service-cert is supported)
+            displayName: Serving Certs Service Name
             path: security.endpointEncryption.certServiceName
           - description: Name of the secrect that contains certificates
+            displayName: Certificates SecretName
```
Comment (Crumby) [inline deploy/olm-catalog/infinispan-operator.clusterserviceversion.yaml]: SecretName -> Secret Name?

File: deploy/olm-catalog/infinispan-operator.clusterserviceversion.yaml (hunk 1 above)
Comment (rigazilla) [reply deploy/olm-catalog/infinispan-operator.clusterserviceversion.yaml]: yeah ok

**Comment by rigazilla** [comment]
merged

**Pull request merged**